### PR TITLE
101 ux topbar

### PR DESCRIPTION
**Pull request by antonyisoardi**
Adding support for basic Foundation Top Bar functionality

Will be adding additional functionality / support later

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,5 @@ public/js
 *.orig
 
 # Ignore everything except for index.html in public folder.
-public/*
\ No newline at end of file
+public/*
+src/pages/test.html
```
Comment (ScottMaclure) [inline .gitignore]: No need for this?

**Comment by ScottMaclure** [comment]
Since the classes in foundation are "top-bar" - I'd rename this entire component from `ux-topbar-container` to `ux-top-bar`, for consistency with Fv5.

Edit: Looking at the example source, I can see why you added the "container" name, but I think you can just hide that detail inside `ux-top-bar`.

File: src/components/ux-topbar-item/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+> ux-topbar-item Documentation
```
Comment (ScottMaclure) [inline src/components/ux-topbar-item/README.md]: Check out the other components - what I do is take Fv5's intro text and then link to the Foundation doc page.

File: .gitignore (hunk 1 above)
Comment (antonyisoardi) [inline .gitignore]: Removed.

File: src/components/ux-topbar-item/README.md (hunk 2 above)
Comment (antonyisoardi) [inline src/components/ux-topbar-item/README.md]: Done this as per ux-top-bar README :)

**Comment by antonyisoardi** [comment]
Updated component to be ux-top-bar instead of ux-topbar to follow Fv5
Component structure follows: 
ux-top-bar > ux-top-bar-container > ux-top-bar-item

Detail should only exist for ux-top-bar so removed README, manifest and use-cases for ux-top-bar-container and ux-top-bar-item

**Comment by ScottMaclure** [comment]
Collapsed the component hierarchy to 2, supports infinitely nested nav items.

**Pull request merged**